find_store.py

```diff
@@ -496,8 +496,11 @@ def price(self, price):
     while first_run or search.daemon:
         if args.clear_console:
             os.system('cls' if os.name == 'nt' else 'clear')
-        search.heb.get_curbside_stores(search)
-        print("Stores with available Curbside (as of {}):\n".format(get_now()))
+        try:
+            search.heb.get_curbside_stores(search)
+            print("Stores with available Curbside (as of {}):\n".format(get_now()))
+        except Exception as e:
+            print("Caught exception: " + str(e))
```

Comment:
Perhaps we should `sleep(3)` and `continue` here. The sleep will give the user a chance to read the exception before continuing the loop and potentially clearing the console. The continue will prevent an unnecessary interval wait and the display of data from the last check.

Reply:
Sounds good.  I didn't think about not waiting a whole interval, as the point here was to ensure that the execution wan't halted, but that is better to just retry quickly.  I don't think anyone will ever actually see the exception with only a 3 second sleep, unless they are looking really intently.